### PR TITLE
[1897] Add course level new page

### PR DESCRIPTION
**Pull request by tvararu**
### Context

Allow providers to select the level of a new course.

### Changes proposed in this pull request

Adds a ontroller action for `continue`, view, page object, and feature spec.

### Guidance to review

Screenshot 2019-08-23 at 12 01 04

### CODE REVIEW

**Comment by tvararu** [comment]
[[M] Select 'Level' (MCFE)](https://trello.com/c/HrhD8qOI/1897-m-select-level-mcfe)

File: app/views/courses/level/_form_fields.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<div class="govuk-form-group ">
+  <div class="govuk-radios govuk-radios" data-module="radios">
+    <% %i[primary secondary further_education].each do |level| %>
```
Comment (fofr) [inline app/views/courses/level/_form_fields.html.erb]: Do we want to pull this list from edit_options/backend?

Comment (tvararu) [reply app/views/courses/level/_form_fields.html.erb]: Not Yet™

Comment (defong) [reply app/views/courses/level/_form_fields.html.erb]: https://ukgovernmentdfe.slack.com/archives/CAHAJFR0X/p1566553607094200

**Pull request merged**